jsonmerge_utils.py

```diff
@@ -5,17 +5,38 @@
 @author: GRENTOR
 """
 import os
+import argparse
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+parser = argparse.ArgumentParser(description="link-check is a broken link identifier")
+parser.add_argument('-ip',
+                    help="Input prefix",
+                    required=True)
+parser.add_argument('-op',
+                    help="Output prefix",
+                    required=True)
+parser.add_argument('-maxFileSize',
+                    help="The maximum file size (in bytes) that each merged file should have",
+                    required=True)
+parser.add_argument('-filepath',
+                    help="Path to directory where all json files are stored",
+                    required=True)
+parser.add_argument('-log_level',
+                    help="The logging level - defaults to INFO [INFO, DEBUG, ERROR]",

```

Comment:
we can remove the last part, since that is covered by choices and metavar :
```suggestion
                    help="The logging level - defaults to INFO",

```